packages/replay/metadata/test.ts

```diff
@@ -18,16 +19,25 @@ const VERSION = 1;
 
 const versions: Record<number, Struct> = {
   1: object({
-    file: optional(string()),
+    file: optional(envString("RECORD_REPLAY_METADATA_TEST_FILE")),
     path: optional(array(string())),
-    result: enums(["passed", "failed", "timedOut"]),
+    result: defaulted(
+      enums(["passed", "failed", "timedOut"]),
+      firstEnvValueOf("RECORD_REPLAY_METADATA_TEST_RESULT")
+    ),
     run: optional(
-      object({
-        id: define("uuid", (v: any) => isUuid.v4(v)),
-        title: optional(string()),
-      })
+      defaulted(
+        object({
+          id: defaulted(
+            define("uuid", (v: any) => isUuid.v4(v)),
+            firstEnvValueOf("RECORD_REPLAY_METADATA_TEST_RUN_ID", "RECORD_REPLAY_TEST_RUN_ID")
+          ),
+          title: optional(envString("RECORD_REPLAY_METADATA_TEST_RUN_TITLE")),
+        }),
+        {}
```

Comment:
What's the purpose of calling defaulted with an empty obj?

Reply:
I couldn't find a way to prepopulate the values within the object from env variables without a default. So, without those defaults, if you called `test.init()` (with no value) you wouldn't get the commit id even if you had the env variable set because it wouldn't try to evaluate the rules within the obj.